app/src/components/notifier/Notifier.js

```diff
@@ -61,8 +60,14 @@ class Notifier extends React.Component {
   }
 }
 
-export function openSnackbar({ message }) {
-  openSnackbarFn({ message });
-}
+const mapStoreToProps = store => ({notify: store.notify});
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
we may not need this afterall... see if you can safely remove it here and in `connect()` call and ensure it doesn't cause trouble in app behavior or testing